mm-go-irckit/service.go

```diff
@@ -263,27 +264,37 @@ func scrollback(u *User, toUser *User, args []string, service string) {
 	}
 
 	if len(args) != 2 {
-		u.MsgUser(toUser, "need SCROLLBACK <channel> <lines>")
+		u.MsgUser(toUser, "need SCROLLBACK (#<channel>|@<user>) <lines>")
 		u.MsgUser(toUser, "e.g. SCROLLBACK #bugs 10 (show last 10 lines from #bugs)")
 		return
 	}
 
 	limit, err := strconv.Atoi(args[1])
 	if err != nil {
-		u.MsgUser(toUser, "need SCROLLBACK <channel> <lines>")
+		u.MsgUser(toUser, "need SCROLLBACK (#<channel>|@<user>) <lines>")
 		u.MsgUser(toUser, "e.g. SCROLLBACK #bugs 10 (show last 10 lines from #bugs)")
 		return
 	}
 
-	if !strings.Contains(args[0], "#") {
-		u.MsgUser(toUser, "need SCROLLBACK <channel> <lines>")
+	if !strings.HasPrefix(args[0], "#") && !strings.HasPrefix(args[0], "@") {
+		u.MsgUser(toUser, "need SCROLLBACK (#<channel>|@<user>) <lines>")
```

Comment:
Okay to use @<user> or should we make this consistent with updatelastviewed? So:

    if strings.Contains(args[0], "#") {
        ...
    } else if updateUser, exists := u.Srv.HasUser(args[0]); exists && updateUser.Ghost {
        ...
    } else {
        print help
    }

Reply:
I'd like to make it consistent with updatelastviewed, the `@user` isn't really native irc

Reply:
It is, when you have ops haha. But yes, will fix and make it consistent. Thanks